spec/features/staff/users_can_search_for_activities_spec.rb

```diff
@@ -2,15 +2,13 @@
   let(:user) { create(:beis_user) }
   before { authenticate!(user: user) }
 
-  let!(:project) { create(:programme_activity, roda_identifier_fragment: "roda-id", title: "Project A") }
-  let!(:third_party_project) { create(:third_party_project_activity, roda_identifier_fragment: "roda-id", title: "Third-party Project B") }
+  let!(:project) { create(:programme_activity, roda_identifier: "roda-id", title: "Project A") }
 
-  scenario "searching by RODA identifier fragment" do
+  scenario "searching by RODA identifier" do
     visit "/"
     fill_in :query, with: "roda-id"
     click_button t("form.activity_search.submit")
 
     expect(page).to have_link project.title, href: organisation_activity_path(project.organisation, project)
-    expect(page).to have_link third_party_project.title, href: organisation_activity_path(third_party_project.organisation, third_party_project)
```

Comment:
From the POV of users, this equates to removing functionality without a workaround, unless we add partial string matching. Are we OK with this?

Reply:
I have been thinking about this, and I don't know. Do we need to keep the fragments around in that case? Is it even useful? I'll raise it with the rest of the team - thanks for picking it up 👍

Reply:
This appears solved by the partial string search you've added, right? I assume it's intentional that you haven't re-added it to the feature spec, and covered it in the unit spec.

Reply:
Yeah, the feature spec just covers the most common use case now